README.md

```diff
@@ -141,6 +136,10 @@ ${DREDD_EXECUTABLE} -p build/compile_commands.json math/src/*.cc --mutation-info
 The `-p` option allows the compilation database generated by CMake above to be passed to Dredd.
 This is so that Dredd knows the correct compiler options to use when processing each source file.
 
+The --mutation-info-file argument is used to specify a JSON file to which Dredd will output 
+machine-readable information about the mutations it applied. This argument is optional and for the 
+purposes of this example, `mutant-info.json` can be ignored.
```

Comment:
Actually it is relevant lower down - see "To learn about the mutants that Dredd has made available ...".

I suggest replacing "This argument is ... can be ignored." with "We explain below how the `mutant-info.json` file created via the `--mutation-info-file` argument can be used to query the mutants that Dredd has introduced."